models/money/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "@commercetools-test-data/money",
+  "version": "4.4.0",
+  "description": "Data model for commercetools API CentPrecisionMoney",
```

Comment:
```suggestion
  "description": "Data model for commercetools API Money",
```
???

Reply:
Rogue copypasta strikes again!

Reply:
Addressed in 588274198e7103b275307fe755d79761bad755e3